test/integration/component/test_ldap_autoImport.py

```diff
@@ -0,0 +1,663 @@
+        self.assertEquals(self.delflag6, 1, "Linking LDAP failed,please check the configuration")
+
+        loginRes = self._checklogin(
+            self.services["configurableData"]["link_ldap_details"]["linkLdapUsername"],
+            self.services["configurableData"]["link_ldap_details"]["linkLdapPassword"],
+            self.domain2.name,
+            method="POST")
+        self.debug(loginRes)
+        self.assertEquals(loginRes, 1, self.reason)
+        #domain 3 ends here
+
+        #deleting domain 2 and 3 here
+        try:
+                deleteDomain2 = deleteDomain.deleteDomainCmd()
+                deleteDomain2.id = self.domainRes2.id
+                deleteDomain2.cleanup = 'true'
+
+                domain_name = self.domainRes2.name
+
+                self.apiClient.deleteDomain(deleteDomain2)
+               # self.domainRes.delete(self.apiclient, cleanup=True)
+                self.debug(
+                    "Deleted the the following account name %s:" %
+                    domain_name)
+        except Exception as e:
+                raise Exception("Warning: Exception during cleanup : %s" % e)
+
+
+
+        try:
+                deleteDomain3 = deleteDomain.deleteDomainCmd()
+                deleteDomain3.id = self.domainRes3.id
+                deleteDomain3.cleanup = 'true'
+
+                domain_name = self.domainRes3.name
+
+                self.apiClient.deleteDomain(deleteDomain3)
+               # self.domainRes.delete(self.apiclient, cleanup=True)
+                self.debug(
+                    "Deleted the the following account name %s:" %
+                    domain_name)
+        except Exception as e:
+                raise Exception("Warning: Exception during cleanup : %s" % e)
+
+        return
+
+    def test_14_ldap(self):
+        #Password/domain empty --> login should fail
+        loginRes = self._checklogin("","",
+                self.services["configurableData"]["link_ldap_details"]["domain_name"],
+                method="POST")
+        self.debug(loginRes)
+        self.assertEquals(loginRes,None, self.reason)
+
+    def _checklogin(cls, username, password, domain, method):
+        """
+
+        :param username:
+        :param password:
+
+        """
+        cls.debug("Attempting to login.")
+
+        try:
+            loginParams = login.loginCmd()
+            loginParams.username = username
+            loginParams.password = password
+            loginParams.domain = domain
+            loginRes = cls.apiClient.login(loginParams, method)
+            cls.debug("login response %s" % loginRes)
+            if loginRes is None:
+                cls.debug("login not successful")
+                return 0
+            else:
+                cls.debug("login successful")
+                return 1
+
+        except Exception as p:
+            cls.debug("login operation failed %s" % p)
+            cls.reason = "Login operation Failed %s" % p
+
+
```

Comment:
@sarathkouk Thanks for processing the feedback. Did you also do the PEP8 on the files?